test/test_reroot.py

```diff
@@ -37,33 +36,38 @@
 class Tests(unittest.TestCase):
     
     def test_reroot_trifurcated_tree_at_longest_child(self):
-        test_tree_1 =TreeNode.read(StringIO(u'(A:0.1,B:0.2,(C:0.3,D:0.4):0.5);'))
-        test_tree_2 =TreeNode.read(StringIO(u'(A:0.5,B:0.2,(C:0.3,D:0.4):0.1);'))
-        test_tree_3 =TreeNode.read(StringIO(u'(A:0.2,B:0.5,(C:0.3,D:0.4):0.1);'))
+        test_tree_1 =Tree.get(schema='newick', data=u'(A:0.1,B:0.2,(C:0.3,D:0.4):0.5);')
+        test_tree_2 =Tree.get(schema='newick', data=u'(A:0.5,B:0.2,(C:0.3,D:0.4):0.1);')
+        test_tree_3 =Tree.get(schema='newick', data=u'(A:0.2,B:0.5,(C:0.3,D:0.4):0.1);')
         
-        expected_test_tree_1 ="((A:0.1,B:0.2):0.25,(C:0.3,D:0.4):0.25)root:0;"
-        expected_test_tree_2 ="((B:0.2,(C:0.3,D:0.4):0.1):0.25,A:0.25)root:0;"
-        expected_test_tree_3 ="((A:0.2,(C:0.3,D:0.4):0.1):0.25,B:0.25)root:0;"
+        expected_test_tree_1 = str(Tree.get(schema='newick', data=u"((C:0.3,D:0.4):0.25,(A:0.1,B:0.2):0.25);"))
+        expected_test_tree_2 = str(Tree.get(schema='newick', data=u"(A:0.25,(B:0.2,(C:0.3,D:0.4):0.1):0.25);"))
+        expected_test_tree_3 = str(Tree.get(schema='newick', data=u"(B:0.25,(A:0.2,(C:0.3,D:0.4):0.1):0.25);"))
         
```

Comment:
Some of these expected trees needed re-arranging, but the overall topology is the same. The clades were just swapped around